LHC/AC_Dipole_Tracking/job.madx

```diff
@@ -74,9 +74,9 @@ if(is_thin==1) {
 ! ============================= !
 
 ! Will be at 0 from turn 0 to 'ramp1', then goes up until max strength at 'ramp2', stays at max strength until 'ramp3', and finally goes down to 0 at 'ramp4' where it will stay until end of tracking.
-! Takes 2000 turns to ramp up AC Dipole (needed to preserve adiabaticity)
+! Usually 2000 ramp turns and 6600 'flat-top' turns for the AC dipole in the LHC, better to reproduce this.
 ! If you don't let the AC dipole ramp down adiabatically Drive will cry and say 'NO KICK FOUND'.
-! Usually 6600 turns of AC dipole 'flat-top' in the LHC, better to reproduce this.
+! Importantly, to see an effect on tracking the AC dipole needs to be installed as a kicker. This does NOT affect TWISS functions: to do so, it should be installed as a matrix element.
```

Comment:
A very kind person would also add a simple example to create an AC-dipole model

Reply:
I guess I'll be very kind in another PR then.